site/content/blog/markdowndb-basics-tutorial-2023.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Learn how to use MarkdownDB with our First Tutorial!
+description: We've just released our first tutorial that covers the fundamentals of MarkdownDB - our new package for treating markdown files as a database. If you've been curious about how to manage your markdown files more effectively, check it out!
+date: 2023-05-26
+author: Ola Rubaj
+---
+
+We've just released our first tutorial that covers the fundamentals of [MarkdownDB](https://github.com/datopian/markdowndb) - our new package for treating markdown files as a database. If you've been curious about how to manage your markdown files more effectively, this tutorial is an excellent starting point!
+
+![Gif](/assets/blog/markdowndb.gif)
+
+[👉 Check it out!](https://github.com/datopian/markdowndb/blob/main/examples/basic-example/README.md)
+
+## What is MarkdownDB?
+
+MarkdownDB can parse your markdown files, extract structured data, such as frontmatter, tags, back- and forward links, and create an index in a local SQLite database. It also provides a lightweight JavaScript API for querying the database and importing files into your application. With MarkdownDB, you have a powerful tool that allows you to efficiently query your markdown data.
+
+[🔍 Click here to learn more!](https://github.com/datopian/markdowndb)
+
+## What you're going to learn
+
+This tutorial guides you through the steps of creating SQLite database of our GitHub projects, using markdown files and MarkdownDB package. In addition, it demonstrates how to write a simple script to query a database for the projects and display them in an attractive format on the terminal.
```

Comment:
This tutorial guides you through the steps of creating a small project catalog. In our case we used our open source projects on github - but you could use anything! We use simple markdown files to describe each project and then display them with the MarkdownDB package.